packages/freerdp.rb

```diff
@@ -3,21 +3,21 @@
 class Freerdp < Package
   description 'FreeRDP is a free implementation of the Remote Desktop Protocol.'
   homepage 'https://www.freerdp.com/'
-  version '3.0.0-a78182c'
+  version '2.10.0-2a72946'
```

Comment:
Downgrade?

Reply:
Yes. Intentional.

Reply:
Only remmina uses freerdp, so it is fine.